### ISSUE TITLE
SEO Landing Pages Current Status

### ISSUE THREAD

**Issue by SteffanThorn** [Giveth/giveth-planning#1556]
I've made this issue of SEO issues assigned to me in one place so it's easier for @surbhiaroraa and relevant people to keep track of where I'm at in the process. 

-How Nonprofits Can Use Crypto for Fundraising #1522 - edited based on feedback
-The Best Crowdfunding Platforms for Nonprofits in 2023 #1541 -- waiting on feedback
-How Can Nonprofits Measure the Success of Their Fundraising #1539 -- Split into a new issue #1555, not sure if I'm to edit the KPI article or not. 
-How to Get Crypytocurrency Donations #1526 -- edited based on feedback 
-Why Crypto is the Future of Raising Money for Charity #1524  -- waiting on feedback
-How Can Giveth help in Your Fundraising #1555  -- waiting on feedback 
-The Best "Crypto" Donation Platforms for Nonprofits for 2023 #1553 -- need to write draft

**Comment by SteffanThorn** [comment]
-The Best "Crypto" Donation Platforms for Nonprofits for 2023 #1553 -- draft provided (with new suggested title of "The Best Crypto Initiatives of 2023")

**Comment by SteffanThorn** [comment]
@oyealmond @danelsuga @ahmadabugosh tagging you in here so it's easier to organize and see where I need help. 

Not sure who is meant to be the sign off for "Ready to publish" (Ahmad or Surbhi?)

**Comment by surbhiaroraa** [comment]
Absolutely @oyealmond @SteffanThorn 
I am reviewing everything today. Planning design inputs and will move to publishing after getting necessary graphics done. 

Thank you @SteffanThorn for all your efforts. Please mention completed against every topic once its completely done from your end to indicate the action is pending on my end. Thank you

**Comment by SteffanThorn** [comment]
@surbhiaroraa I believe I'm caught up with all comments. Have resolved them. 

For #1555, the last project "DAODrops" I need to confirm one of the cons. 

For other articles I've added my own comments where I think you should be able to resolve.

**Comment by oyealmond** [comment]
Hey team. @surbhiaroraa @SteffanThorn @danelsuga @ahmadabugosh 
We have 6 landing pages ready to publish & follow up. (Issue https://github.com/Giveth/giveth-planning/issues/1539, was replaced with https://github.com/Giveth/giveth-planning/issues/1555)

- https://github.com/Giveth/giveth-planning/issues/1522 [text here](https://docs.google.com/document/d/1Agtv3psOUSA_He0cxVtSKba-8YjkdKIK7lyMnCelJHY/edit#heading=h.yw3retihq4me)
- https://github.com/Giveth/giveth-planning/issues/1541 [text here](https://docs.google.com/document/d/1guibAqKdJuJEjoRJqvwVy2BRRyMDBZKfDgnNMLc1CcM/edit#heading=h.9p0nvm5coqns)
- https://github.com/Giveth/giveth-planning/issues/1524 [text here](https://docs.google.com/document/d/1WK9rPuRIKJEzZdP5oOY8j1xUxPeYNilAE825uAeiAI4/edit)
- https://github.com/Giveth/giveth-planning/issues/1555 [text here](https://docs.google.com/document/d/1Y2DQ1bksM7w0hdFK0uk6Tr9d7D0zoa4aafPLcc8RkVw/edit#heading=h.451tn2st2b4a)
- https://github.com/Giveth/giveth-planning/issues/1526 [text here](https://docs.google.com/document/d/1rRYU88Dcjz2p2tunf2CLd8pw9uzDQCgBjKC7pdDNd7o/edit)
- https://github.com/Giveth/giveth-planning/issues/1553 [text here](https://docs.google.com/document/d/11Tq6EK0k20nJsfBNQqIIA2gxPg_akDXFk09Oen50yDg/edit)

**Comment by SteffanThorn** [comment]
I think we could combine them with a paid ad strategy on both Facebook and X to increase exposure.

As you said @oyealmond it's very intro level friendly and FB has a TON of nonprofits -- so I definitely see an importance of focusing on paid ads over there.

I can join forces with @surbhiaroraa (??and @ahmadabugosh??) to produce these ads and test them on FB.

**Comment by oyealmond** [comment]
hey @SteffanThorn, I believe there will be a paid campaign effort for this yes! Here's the related issue https://github.com/Giveth/giveth-planning/issues/1596#issuecomment-1752401198. I'll suggest to coordinate w @ahmadabugosh and @surbhiaroraa

**Comment by oyealmond** [comment]
All Landing pages are now published on Medium. Thank you all!
